### PR TITLE
Chatbot csrf fix

### PR DESCRIPTION
**Pull request by shefalinayak**
## Description
Related to [this issue](https://github.com/department-of-veterans-affairs/covid19-chatbot/issues/227)

- Removed call to /status for CSRF
- Encapsulates chatbot in a React component
- Waits for feature flag loading to be false before rendering chatbot
- Renders loading component until requests (feature flag & token) are complete

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/coronavirus-chatbot/index.js [hunk 1]
```diff
@@ -139,16 +132,23 @@ export const requestChatBot = async loc => {
...
+    });
     return initBotConversation(token);
   } catch (error) {
     Sentry.captureException(error);
     recordEvent({
       event: `${GA_PREFIX}-connection-failure`,
       'error-key': 'XX_failed_to_init_bot_convo',
     });
-    return null;
+    return error;
```
Comment (shaunhubbard-va) [inline src/applications/coronavirus-chatbot/index.js:151]:
```suggestion
    throw error;
```

File: src/applications/coronavirus-chatbot/components/ChatbotWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+export class ChatbotWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chatbotError: false,
+    };
+  }
+
+  async componentDidMount() {
+    const initialComponent = document.querySelector(`#${idString}`);
```
Comment (shaunhubbard-va) [inline src/applications/coronavirus-chatbot/components/ChatbotWrapper.jsx:19]: If it were me, I would refactor this method to be more readable, but not a show stopper

**Review by ghost — APPROVED** [review]
LGTM, please address the one PR comment about throwing an error vs returning one

File: src/applications/coronavirus-chatbot/components/ChatbotWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      Sentry.captureException(err);
+      recordEvent({
+        event: `${GA_PREFIX}-connection-failure`,
+        'error-key': 'XX_failed_to_start_chat',
+      });
+      recordEvent({
+        event: `${GA_PREFIX}-load-failure`,
+        'error-key': undefined,
+      });
+      // eslint-disable-next-line react/no-did-mount-set-state
```
[the remark this answers is not in the document]
Comment (katieho-va) [reply src/applications/coronavirus-chatbot/components/ChatbotWrapper.jsx]: Same error in renamed file below

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**